modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -543,6 +595,7 @@ public void testPutCustomObjectEntryWithNestedSystemObjectEntry()
 		// Many to many
 
 		_testPutCustomObjectEntryWithNestedSystemObjectEntry(
+			null, null,
```

Comment:
I think it makes no sense having more tests, as the one with the custom field should cover the case without custom field, right?